src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -230,7 +232,7 @@ function MoneyRequestPreview(props) {
                 <View style={[styles.moneyRequestPreviewBox, isScanning || props.isWhisper ? styles.reportPreviewBoxHoverBorder : undefined, ...props.containerStyles]}>
                     {hasReceipt && (
                         <ReportActionItemImages
-                            images={[ReceiptUtils.getThumbnailAndImageURIs(props.transaction.receipt.source, props.transaction.filename || '')]}
+                            images={receiptImages}
```

Comment:
I couldn't repro this crash, but I pushed this change anyway. I added a `hasReceipt` check above instead since we should still avoid complex inline JSX statements like this.